lib/sufia/export/converter.rb

```diff
@@ -0,0 +1,21 @@
+module Sufia
+  module Export
+    # a base class to convert an ActiveFedora object that contains permissions and allow for pretty json.
+    #
+    class Converter
+      # overrides to_json to optionally allow for a pretty version of the json to be outputted
+      #
+      # @param [Boolean] pretty pass true to output formatted json using pretty_generate
+      def to_json(pretty = false)
+        return super unless pretty
+        JSON.pretty_generate(JSON.parse(to_json))
```

Comment:
@mjgiarlo I fooled around with some different ways to call super and `super()` works `super` does not it gives a 
```
     TypeError:
       can't dup TrueClass
```
The result is the same either way.  Which is easier to understand? recalling to_json with no parameters or calling super()?